packages/nx/presets/npm.json

```
@@ -8,7 +8,7 @@
     }
   },
   "pluginsConfig": {
-    "@nrwl/js": {
+    "@nx/js": {
       "analyzeSourceFiles": false
     }
```

Do you think it would be worth having both? E.g.:

```suggestion
    "@nx/js": {
      "analyzeSourceFiles": false
    },
    "@nrwl/js": {
      "analyzeSourceFiles": false
    }
```

I'm not sure that it would - we control the key we read too so its likely fine. Just a thought.

---

Nah, we control it so just one is okay.